src/CommunityToolkit.Maui.UnitTests/Converters/ImageResourceConverter_Tests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using CommunityToolkit.Maui.Converters;
+using CommunityToolkit.Maui.UnitTests.Mocks;
+using Microsoft.Maui.Controls;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Converters;
+
+public class ImageResourceConverter_Tests : BaseTest
+{
+	public ImageResourceConverter_Tests() : base()
+	{
+		Application.Current = new MockApplication();
+	}
+
+	protected override void Dispose(bool isDisposing)
+	{
+		Application.Current = null;
+
+		base.Dispose(isDisposing);
+	}
+
+	[Fact]
+	public void ImageResourceConverter()
+	{
+		const string resourceToLoad = "CommunityToolkit.Maui.UnitTests.Resources.dotnet-bot.png";
+
+		var expectedResource = ImageSource.FromResource(resourceToLoad);
+		var expectedMemoryStream = GetStreamFromImageSource(expectedResource);
+
+		var imageResourceConverter = new ImageResourceConverter();
+		var result = (ImageSource)imageResourceConverter.Convert(resourceToLoad, typeof(ImageResourceConverter), null, CultureInfo.CurrentCulture);
+
+		Assert.True(StreamEquals(GetStreamFromImageSource(result), expectedMemoryStream));
+	}
+
+	[Fact]
+	public void NullValueReturnsNull()
+	{
+		var imageResourceConverter = new ImageResourceConverter();
+
+		Assert.Null(imageResourceConverter.Convert(null, typeof(ImageResourceConverter), null, CultureInfo.CurrentCulture));
+	}
+
+	[Fact]
+	public void ThrowsIfNotAString()
```

Comment:
@brminnick What is the preference for using multiple Asserts versus making this test a Theory with class data?  (just a question, cause I'm a n00b here)

Reply:
Mostly it was just me being lazy and copy/pasting your Assert statement 😄

You're welcome to refactor it into `Theory` + `MemberData` like we've done here for `ListIsNotNullOrEmptyConverter_Tests.cs`:
https://github.com/CommunityToolkit/Maui/blob/6767dd092da74344eec63c33dc706b2e11f05859/src/CommunityToolkit.Maui.UnitTests/Converters/ListIsNotNullOrEmptyConverter_Tests.cs#L12-L29

Reply:
Gotcha - I think I will :)

Reply:
Done